src/features/controls.ts

```diff
@@ -54,9 +54,12 @@ export const DEFAULT_CAMERA_ORBIT = '0deg 75deg 105%';
 const DEFAULT_CAMERA_TARGET = 'auto auto auto';
 const DEFAULT_FIELD_OF_VIEW = 'auto';
 
-export const fieldOfViewIntrinsics = {
-  basis: [numberNode(DEFAULT_FOV_DEG * Math.PI / 180, 'rad')],
-  keywords: {auto: [null]}
+export const fieldOfViewIntrinsics = (element: ModelViewerElementBase) => {
+  return {
+    basis: [numberNode(
+        (element as any)[$idealFieldOfView] * Math.PI / 180, 'rad')],
```

Comment:
Why not skip the pi/180 and just call it degrees?

Reply:
All numbers in the style evaluator are currently normalized to either radians or meters. We can change that, but I would rather change it later.

Reply:
gotcha, I missed that.